tests/kafkatest/services/kafka/kafka.py

```diff
@@ -239,12 +239,12 @@ def start_node(self, node):
 
         self.start_jmx_tool(self.idx(node), node)
         if len(self.pids(node)) == 0:
-            raise Exception("No process ids recorded on node %s" % str(node))
+            raise Exception("No process ids recorded on node %s" % node.account.hostname)
 
     def pids(self, node):
         """Return process ids associated with running processes on the given node."""
         try:
-            cmd = "jcmd | grep -e kafka.Kafka | awk '{print $1}'"
+            cmd = "jcmd | grep -e kafka.Kafka -e metrics.SupportedKafka | awk '{print $1}'"
```

Comment:
What was the problem with the original version exactly? What behavioral problems was it causing in the tests?